packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1719,6 +1738,26 @@ export const components = (
         },
       },
     },
+    MuiPaper: {
+      styleOverrides: {
+        // depth/menu
+        elevation1: {
+          boxShadow: "0px 1px 2px 0px rgba(39, 39, 39, 0.07)",
+        },
+
+        // depth/dialog
+        elevation2: {
+          boxShadow:
+            "0px 1px 4px 0px rgba(39, 39, 39, 0.08), 0px 4px 6px 0px rgba(39, 39, 39, 0.01), 0px 5px 15px 0px rgba(39, 39, 39, 0.05)",
+        },
+
+        // depth/drawer
+        elevation3: {
+          boxShadow:
+            "0px 1px 4px 0px rgba(39, 39, 39, 0.08), 0px 4px 10px 0px rgba(39, 39, 39, 0.08), 0px 8px 30px 0px rgba(39, 39, 39, 0.10)",
+        },
```

Comment:
Don't we want to use `rem` for these values? Also, why are the colors not in the design tokens? That will make it difficult to override.